examples/tutorials/advance_model_config/01_microbatch.ipynb

```diff
@@ -0,0 +1,319 @@
+{
```

Comment:
<p>Use the same model, as <code>03_tf</code> tutorial: this way it would be easier to compare time of training with different settings (microbatch, multiple devices, etc). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>